Source/WebKit/Platform/SharedMemory.h

```diff
@@ -151,4 +156,14 @@ class SharedMemory : public ThreadSafeRefCounted<SharedMemory> {
 #endif
 };
 
+} // namespace WebKit
+
+namespace IPC {
+
+template<> struct ArgumentCoder<WebKit::SharedMemoryHandle, void> {
+    static void encode(Encoder&, const WebKit::SharedMemoryHandle&);
```

Comment:
Even before we change the `mutable` aspect of `SharedMemoryHandle`, I suggest we follow up by removing this function and making changes everywhere to get things working. Maybe that’s exactly what you mean by getting rid of the need for `mutable`. To state the obvious, the real problem here is not the use of `mutable`, but rather than lack of clarity about what happens to these handles during the encoding process.